features/yoroi-transfer.feature

```diff
@@ -57,10 +57,4 @@ Feature: Transfer Yoroi Wallet funds
     Then I transfer some Ada out of the source wallet
     | fromAddress                                                 | amount     |   
     | Ae2tdPwUPEYx2dK1AMzRN1GqNd2eY7GCd7Z6aikMPJL3EkqqugoFQComQnV | 1000000000 |
-    When I confirm Yoroi transfer funds
-    Then I should see wallet changed notice
-    And I should see on the Yoroi transfer summary screen:
-    | fromAddress                                                 | amount           |   
-    | Ae2tdPwUPEYx2dK1AMzRN1GqNd2eY7GCd7Z6aikMPJL3EkqqugoFQComQnV | 1233567898765    |
-    When I confirm Yoroi transfer funds
-    Then I should see the Yoroi transfer success screen
+    Then I should see the Create wallet screen
```

Comment:
I don't quite understand this change...

Reply:
oh my bad I didn't mean to commit this. This is meant to add a step that can never happen that way the test deadlocks (that way I can easily let the test run up to this point then investigate myself)

Reply:
OK np. I'll take care of this.